portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,16 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random hidden talent to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I know over 100 digits of pi!', 'I play piano', 'I can play French Horn', 'I can play guitar', 'I can play saxophone', 'I can speak a little Spanish!'];
```

Comment:
In JavaScript, stick to 80 characters per line. This particular object will work nicely if you keep `const facts =` on the first line, then put each array member on its own line followed by commas and indented 2 spaces, then close the array on another line with no indent. Let me know if you have questions about that.